packages/api/src/webhooks/cannon.ts

```diff
@@ -546,3 +546,13 @@ export default class WebhookCannon {
     );
   }
 }
+
+export async function addSignatureHeader(
+  params: any,
```

Comment:
This should not be `any`, there is a [very specific type for this](https://github.com/livepeer/livepeer-com/blob/88236319556da6cf0c1c4f0dc9fa3b89d9c87a0c/packages/api/src/util.ts#L2). We should never use `any` unless there is no choice (e.g. third-party lib, too annoying type, idk)